frontend/packages/console-app/src/actions/providers/index.ts

```diff
@@ -9,3 +9,4 @@ export * from './replicaset-provider';
 export * from './replication-controllers-provider';
 export * from './pod-provider';
 export * from './perspective-state-provider';
+export * from './default-provider';
```

Comment:
This file will be removed as part of #14683 by @logonoff 

We should avoid using barrel files (index files that just re-export stuff) for exposed modules, as they may cause unnecessary code to be loaded.

As per #14683 I'd suggest to add the following shared module into `frontend/packages/console-app/package.json`
```json
"defaultProvider": "src/actions/providers/default-provider.ts",
```
and use it in `frontend/packages/console-app/console-extensions.json`
```diff
-"provider": { "$codeRef": "actions.useDefaultActionsProvider" }
+"provider": { "$codeRef": "defaultProvider.useDefaultActionsProvider" }
```

Reply:
I see this file is currently removed.